src/_posts/platform/cli/2000-01-01-features.md

```diff
@@ -1,10 +1,19 @@
 ---
 title: Features
-modified_at: 2019-08-26 00:00:00
+modified_at: 2022-11-09 00:00:00
 tags: cli interface app
 index: 3
 ---
 
+## Check the logged in user
+`scalingo self|whoami`
+
+```bash
+# Return the logged in user with either command
+scalingo self
+scalingo whoami
+```
+
```

Comment:
suggestion: I'd move this command down in the list. I don't know where come the command order but this command seems not relevant enough to be the first shown, WDYT?